src/main.rs

```diff
@@ -113,6 +116,9 @@ fn find_ck3_directory() -> Option<PathBuf> {
 }
 
 fn main() -> Result<()> {
+    #[cfg(windows)]
+    let _ = ansi_term::enable_ansi_support();
```

Comment:
I tried to look up this enable_ansi_support function to see why the return value was being thrown away, but I couldn't find it in the docs. Do you have a reference?

Reply:
It returns an Error, I guess unwrapping it would be nicer.

[LINK TO CODE](https://github.com/ogham/rust-ansi-term/blob/ff7eba98d55ad609c7fcc8c7bb0859b37c7545cc/src/windows.rs#L10)